exporter/prometheusexporter/prometheus_test.go

```diff
@@ -48,6 +48,7 @@ func TestPrometheusExporter(t *testing.T) {
 					"code0": "one0",
 				},
 				Endpoint:         ":8999",
+				MetricsPath:      "/metrics",
```

Comment:
Can we add a test using a different path to ensure the config is being read?

Reply:
Hi, done!